CONTRIBUTE.md

```diff
@@ -0,0 +1,49 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make/new feature you want to add via issue, or any other method with the owners of this repository. 
```

Comment:
I know you got this from the template but I'm not sure we should include this. A lot of good PRs we have received have not been previously discussed and I think enforcing this is to not promote creativity. I'd rather not enforce it.

Any thoughts? @desyncr @jedahan

Reply:
Sure, should I go ahead and take it off? Do you want to replace it with something else?

Reply:
I'd prefer taking it off if that's ok with @jedahan and @desyncr

Reply:
I have added a link to the plugin wiki as you called out earlier.